toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -42,7 +42,13 @@ func doCustomizations(buildDir string, baseConfigPath string, config *imagecusto
 		return err
 	}
 
-	err = addRemoveAndUpdatePackages(buildDir, baseConfigPath, &config.SystemConfig, imageChroot, rpmsSources, useBaseImageRpmRepos)
+	err = refreshInitrd(partitionsCustomized, imageChroot)
```

Comment:
suggestion - isn't it better to check `paritionsCustomized` here and keep `refreshInitrd` implementation generic?